OFMetadataToStash.ps1

```diff
@@ -328,21 +311,23 @@ else {
                     if ($null -ne $StashDBQueryResult){
 
                         #Creating the title we want for the media
-                        $title = "$performername - $creationdatefromOF"
-
+                        $title = "$performername - $OFDBapiType - $OFDBpaidOrFree - $creationdatefromOF"
+						
                         #Quick check to see if this file already has metadata from this script
                         #Only Videos can have details (for now) so we have a condition to check for that here
-                        if (($StashDBQueryResult.title -ne $title) -or (($mediatype -eq "video") -and ($StashDBQueryResult.details -ne $OFDBMedia.text))){

```

Comment:
This failed to add metadata for stores and messages because they don't have text